platform-api/che-core-api-workspace/src/test/resources/stacks.json

```diff
@@ -0,0 +1,142 @@
jYgMiwtMy4zIDQuOSwtNS45IDguOCwtNy44IDMuOSwtMS45IDkuMywtMi44IDE2LjIsLTIuOCA4LjQsMCAxNC45LDEuNyAxOS4yLDUuMiA0LjQsMy41IDcsOC45IDcuOCwxNi41IGwgLTE5LjIsMS4zIEMgMTAyLDMxLjEgMTAxLDI4LjcgOTkuMywyNy4yIDk3LjcsMjUuNyA5NS40LDI1IDkyLjYsMjUgYyAtMi40LDAgLTQuMiwwLjUgLTUuNCwxLjYgLTEuMiwxLjEgLTEuOCwyLjUgLTEuOCw0IDAsMS4yIDAuNSwyLjIgMS41LDMuMSAxLDEgMy4zLDEuOCA2LjgsMi42IDguOSwyLjEgMTUuMiw0LjMgMTkuMSw2LjUgMy44LDIuMiA2LjcsNC44IDguNCw4LjEgMS43LDMuMiAyLjYsNi44IDIuNiwxMC44IDAsNC42IC0xLjIsOC45IC0zLjYsMTIuOSAtMi40LDMuOSAtNS42LDYuOSAtOS44LDguOSAtNC4yLDIgLTkuNSwzIC0xNS45LDMgLTExLjIsMCAtMTksLTIuNCAtMjMuMiwtNy4yIC00LC00LjYgLTYuNSwtMTAuNiAtNy4xLC0xOCBsIDAsMCAwLDAgeiBtIDU4LjgsLTQ4LjcgMzMuOCwwIGMgNy40LDAgMTIuOSwxLjkgMTYuNiw1LjggMy42LDMuOCA1LjUsOS40IDUuNSwxNi41IDAsNy40IC0yLDEzLjEgLTYsMTcuMiAtNCw0LjEgLTEwLjEsNi4yIC0xOC4zLDYuMiBsIC0xMS4xLDAgMCwyNy4xIC0yMC41LDAgMCwtNzIuOCAwLDAgeiBtIDIwLjQsMzEgNSwwIGMgMy45LDAgNi43LC0wLjcgOC4zLC0yLjMgMS41LC0xLjUgMi40LC0zLjUgMi40LC01LjcgMCwtMi4yIC0wLjcsLTQuMyAtMi4xLC01LjggLTEuNCwtMS41IC0zLjksLTIuNCAtNy43LC0yLjQgbCAtNS44LDAgMCwxNi4yIC0wLjEsMCAwLDAgeiIKICAgICAgICAgaWQ9InBhdGgzMDQwIiAvPgogICAgICA8cGF0aAogICAgICAgICBkPSJtIDE4Nyw4NC42IDAsLTYyLjEgOC4zLDAgMCw4LjggYyA0LC02LjggOS44LC0xMC4yIDE3LjMsLTEwLjIgMy4zLDAgNi4zLDAuNiA5LDIgMi43LDEuNCA0LjgsMy4xIDYuMiw1LjMgMS40LDIuMiAyLjMsNC43IDIuOSw3LjcgMC40LDEuOSAwLjUsNS40IDAuNSwxMC4yIGwgMCwzOC4xIC05LjIsMCAwLC0zNy43IGMgMCwtNC4zIC0wLjQsLTcuNSAtMS4xLC05LjYgLTAuNywtMi4xIC0yLC0zLjggLTMuOCwtNS4xIC0xLjgsLTEuMyAtNCwtMS45IC02LjUsLTEuOSAtMy45LDAgLTcuMywxLjUgLTEwLjIsNC4zIC0yLjgsMi44IC00LjMsOC4yIC00LjMsMTYuMiBsIDAsMzMuOSAtOS4xLDAuMSAwLDAgeiBtIDk1LjUsLTIwIDkuNSwxLjQgYyAtMS41LDYuNCAtNC4zLDExLjMgLTguMywxNC44IC00LjEsMy41IC05LjMsNS4zIC0xNS42LDUuMyAtNy45LDAgLTE0LjIsLTIuOCAtMTguOSwtOC40IC00LjYsLTUuNiAtNywtMTMuNCAtNywtMjMuNSAwLC0xMC41IDIuNCwtMTguNSA3LC0yNC4zIDQuNywtNS43IDEwLjgsLTguNyAxOC4zLC04LjcgNy4zLDAgMTMuMiwyLjggMTcuOCw4LjUgNC42LDUuNyA2LjksMTMuNiA2LjksMjMuOCAwLDAuNiAwLDEuNSAtMC4xLDIuOCBsIC00MC42LDAgYyAwLjQsNi44IDIsMTIgNS4xLDE1LjcgMywzLjYgNi44LDUuNSAxMS40LDUuNSAzLjQsMCA2LjMsLTEgOC43LC0zIDIuNSwtMi4yIDQuNCwtNS41IDUuOCwtOS45IGwgMCwwIDAsMCB6IG0gLTMwLjMsLTE3IDMwLjQsMCBjIC0wLjQsLTUuMiAtMS41LC05LjEgLTMuNSwtMTEuOCAtMi45LC00IC02LjcsLTYuMSAtMTEuNCwtNi4xIC00LjIsMCAtNy43LDEuNiAtMTAuNyw0LjggLTIuOSwzLjMgLTQuNCw3LjcgLTQuOCwxMy4xIGwgMCwwIDAsMCB6IG0gNzEuNSwyNy42IDEuNCw5LjMgYyAtMi42LDAuNiAtNC45LDAuOSAtNi45LDAuOSAtMy40LDAgLTUuOSwtMC42IC03LjcsLTEuOCAtMS44LC0xLjIgLTMuMSwtMi44IC0zLjksLTQuNyAtMC43LC0xLjkgLTEuMSwtNi4xIC0xLjEsLTEyLjQgbCAwLC0zNS43IC0yOS41LDAgMCwtOC4yIDI5LjQsMCAwLC0xNS40IDkuMiwtNi4zIDAsMjEuNyA5LjMsMCAwLDguMiAtOS4zLDAgMCwzNi4yIGMgMCwzIDAuMiw0LjkgMC41LDUuNyAwLjQsMC44IDAuOCwxLjUgMS41LDIuMSAwLjcsMC41IDEuOCwwLjcgMy4xLDAuNyAxLDAuMSAyLjQsMCA0LC0wLjMgbCAwLDAgMCwwIHoiCiAgICAgICAgIGlkPSJwYXRoMzA0MiIgLz4KICAgICAgPHBhdGgKICAgICAgICAgZD0ibSAxNzMuMyw2OC41IGMgNCwwIDcuNCwzLjQgNy40LDcuNSAwLDQuMSAtMy4zLDcuNSAtNy40LDcuNSAtNCwwIC03LjQsLTMuNCAtNy40LC03LjUgMCwtNC4xIDMuMywtNy41IDcuNCwtNy41IGwgMCwwIDAsMCB6IgogICAgICAgICBpZD0icGF0aDMwNDQiIC8+CiAgICA8L2c+CiAgPC9nPgo8L3N2Zz4K"
```

Comment:
Use resource reference here, what if i what to add a new icon, what should i do?